src/WebJobs.Script.WebHost/Models/AzureStorageInfoValue.cs

```diff
@@ -83,7 +83,12 @@ public static AzureStorageInfoValue FromEnvironmentVariable(KeyValuePair<string,
             }
 
             var parts = environmentVariable.Value?.Split(new[] { '|' }, StringSplitOptions.RemoveEmptyEntries);
-            if (parts == null || parts.Length != 4)
+            if (parts == null)
+            {
+                return null;
+            }
+
+            if (parts.Length != 4 && parts.Length != 5)
```

Comment:
what's this condition?  Based on the condition, this will never execute the block?